cloudslang-cli/src/main/resources/log4j.properties

```diff
@@ -4,7 +4,7 @@ def.files.backup.count=10
 log4j.rootLogger=INFO, A1
 
 log4j.appender.A1=org.apache.log4j.RollingFileAppender
-log4j.appender.A1.File=logs/execution.log
+log4j.appender.A1.File=${user.dir}/target/cslang/logs/execution.log
```

Comment:
e.g. `getClass().getResource("")` returns a STR tat contains path to `lib` folder - you may be able to use this (please check the other possibilities, there may be better ones)